src/components/Dialogs/BulkRenameFilesDialog.vue

```diff
@@ -0,0 +1,317 @@
+      close()
+    })
+}
+
+const close = () => {
+  isOpened.value = false
+}
+
+watch([regexpInput, regexpFlagsInput, targetInput], () => {
+  dryRunRename()
+})
+
+onMounted(() => {
+  parseNode(props.node)
+})
+</script>
+
+<template>
+  <v-dialog v-model="isOpened" persistent>
+    <v-card density="compact">
+      <v-card-title>
+        {{ $t('dialogs.bulkRenameFiles.title') }}
+        <v-btn style="float: right" icon="mdi-close" variant="plain" density="compact" @click="close()"></v-btn>
+      </v-card-title>
+      <v-card-text>
+        <v-form v-model="isFormValid" ref="form">
+          <v-text-field hide-details density="compact" v-model="regexpInput" :rules="rules" :label="$t('dialogs.bulkRenameFiles.regexp')">
+            <template v-slot:append>
+              <v-select
+                class="flags-select"
+                v-model="regexpFlagsInput"
+                :items="['d', 'g', 'i', 'm', 's', 'u', 'v', 'y']"
+                hint="Select Regular Expression Flags"
+                label="Flags"
+                density="compact"
+                multiple
+                hide-details></v-select>
+            </template>
+          </v-text-field>
+          ->
+          <v-text-field hide-details density="compact" v-model="targetInput" :rules="rules" :label="$t('dialogs.bulkRenameFiles.target')" />
+          <v-badge color="success" :content="candidateItems.length">
+            <v-btn :loading="running" :disabled="!isFormValid || hasDuplicated" color="primary" @click="run()">{{ $t('dialogs.bulkRenameFiles.run') }}</v-btn>
+          </v-badge>
+        </v-form>
+        <v-data-table-virtual :headers="headers" :items="items" density="compact" height="calc(100vh - 200px)" fixed-header>
+          <template v-slot:item="{ index, item }">
+            <v-data-table-row v-show="item.show" :index="index" :item="item as any">
+              <template v-slot:item.selected>
+                <v-checkbox-btn v-if="item.type === 'file'" v-model="item.selected" :color="item.targetName && 'indigo'" @change="fileCheckChange(item)" />
+                <v-checkbox-btn v-else v-model="item.selected" :indeterminate="item.indeterminate" @change="folderCheckChange(item)" />
+              </template>
+              <template v-slot:item.name>
+                <span class="fold-toggle" :class="{ clickable: item.type === 'folder' }" @click="item.type === 'folder' && toggleFolderFolded(item, !item.folded)">
+                  <v-tooltip v-if="item.type === 'folder'" location="top" activator="parent">
+                    {{ t(`dialogs.bulkRenameFiles.${item.folded ? 'unfold' : 'fold'}`) }}
+                  </v-tooltip>
+                  {{ '&emsp;'.repeat(item.indent) }}
+                  <v-icon v-if="item.type === 'folder'">{{ item.folded ? 'mdi-chevron-down' : 'mdi-chevron-up' }}</v-icon>
+                  <v-icon v-if="node.fullName === ''" icon="mdi-file-tree" />
```

Comment:
When opening the dialog through the root element, all icons are `mdi-file-tree`. It works properly when opening it through a folder.

```suggestion
                  <v-icon v-if="item.fullName === ''" icon="mdi-file-tree" />
```